guidelines/index.html

```diff
@@ -967,7 +967,7 @@ <h3 id="x0-2-wcag-2-layers-of-guidance"><span class="secno">0.2 </span>WCAG 2 
 					</li>
 				</ul>
 				<p>このガイダンスのレイヤー (原則、ガイドライン、達成基準、十分な達成方法及び参考達成方法) はすべて、コンテンツをよりアクセシブルにする方法に関するガイダンスを提供するために連携している。コンテンツ制作者は可能な範囲で最も広い利用者のニーズに最大限対処できるように、参考達成方法を含めたすべてのレイヤーを確認して適用することが推奨される。</p>
-				<p>注意すべきなのは、最高レベル (AAA) で適合しているコンテンツでさえも、すべての種類、程度あるいは組合せの障害のある個人、特に、認知、言語及び学習の面において、アクセシブルではないということである。コンテンツ制作者は、参考達成方法を含むすべての達成方法を考慮するとともに、 ウェブコンテンツが可能な限りアクセシブルであることを確実にするためにも、こうした人たちに対して、現状における最善の対応策に関するアドバイスを追い求めることが推奨される。<a href="https://www.w3.org/WAI/WCAG21/Understanding/understanding-metadata">メタデータ</a>は、彼らのニーズに最適なコンテンツを探し出す際に、利用者を支援できる可能性がある。 </p>
+				<p>注意すべきなのは、最高レベル (AAA) で適合しているコンテンツでさえも、すべての種類、程度あるいは組合せの障害のある個人、特に、認知、言語及び学習の面において、アクセシブルではないということである。コンテンツ制作者は、参考達成方法を含むすべての達成方法を考慮するとともに、ウェブコンテンツが可能な限りアクセシブルであることを確実にするためにも、こうした人たちに対して、現状における最善の対応策に関するアドバイスを追い求めることが推奨される。<a href="https://www.w3.org/WAI/WCAG21/Understanding/understanding-metadata">メタデータ</a>は、彼らのニーズに最適なコンテンツを探し出す際に、利用者を支援できる可能性がある。 </p>
```

Comment:
ついでに末尾の空白も消しておきましょう？

```suggestion
				<p>注意すべきなのは、最高レベル (AAA) で適合しているコンテンツでさえも、すべての種類、程度あるいは組合せの障害のある個人、特に、認知、言語及び学習の面において、アクセシブルではないということである。コンテンツ制作者は、参考達成方法を含むすべての達成方法を考慮するとともに、ウェブコンテンツが可能な限りアクセシブルであることを確実にするためにも、こうした人たちに対して、現状における最善の対応策に関するアドバイスを追い求めることが推奨される。<a href="https://www.w3.org/WAI/WCAG21/Understanding/understanding-metadata">メタデータ</a>は、彼らのニーズに最適なコンテンツを探し出す際に、利用者を支援できる可能性がある。</p>
```